Source/WebCore/platform/graphics/gstreamer/VideoFrameGStreamer.cpp

```diff
@@ -508,6 +508,12 @@ RefPtr<ImageGStreamer> VideoFrameGStreamer::convertToImage()
     return convertSampleToImage(m_sample);
 }
 
+Ref<VideoFrame> VideoFrameGStreamer::clone()
+{
+    // FIXME: Implement clone.
+    return *this;
```

Comment:
`createWrappedSample(m_sample, presentationTime())` should work, hopefully.

Reply:
Done